lib/geckoboard.rb

```diff
@@ -10,6 +10,14 @@
 require 'geckoboard/payload_formatter'
 require 'geckoboard/errors'
 
+require 'geckoboard/field'
+require 'geckoboard/number'
+require 'geckoboard/date'
+require 'geckoboard/date_time'
+require 'geckoboard/money'
+require 'geckoboard/percentage'
+require 'geckoboard/string'
```

Comment:
If it was me writing this, I'd keep them all in one file. I don't feel too strongly about it, but it might just be easier to have `geckoboard/field_types` or something.